### DISCUSSION TITLE
lock edit conditionally.

### DISCUSSION THREAD

**Discussion by itrixinc** [filamentphp/filament#3966] [General]
I am a newbie to laravel + filament.

I need a way to lock complete editing based on field status.
eg. if field freeze == true, lock everything, no save allowed.

I tried to disable complete $form
            ->schema([
...
])->disabled(fn(Closure $get) => $get('freeze') === true))

but I get an error 
	Expected type 'bool'. Found 'Closure'

Can you suggest the best way to handle this?

**Comment by danharrin** [comment]
You cannot lock the form like that, because `$get` is only available inside the schema

So, you should wrap the fields in a `Group` or `Grid` and pass `->disabled()` there

**Reply by itrixinc** [reply]
Thanks. That worked.

But save/delete buttons on page/relationshipManager listing are active, How can those be disabled conditionally?
